combat-trainer.lic

```diff
@@ -702,6 +716,9 @@ class SpellProcess
     @necromancer_healing = settings.necromancer_healing
     echo("  @necromancer_healing: #{@necromancer_healing}") if $debug_mode_ct
 
+    @necromancer_zombie = settings.waggle_sets['zombie']
```

Comment:
Is this still a waggle set?

Reply:
Yea

Reply:
Should it not be?

Reply:
Sorry, I misread something. You're good.